mlflow/server/js/src/experiment-tracking/components/ExperimentRunsTableMultiColumnView2.test.js

```diff
@@ -84,6 +86,41 @@ describe('ExperimentRunsTableMultiColumnView2', () => {
     expect(output).toEqual('-');
   });
 
+  test('should render logged model link with valid URL', () => {
```

Comment:
@hctpbl I pushed a commit to add this test.

Reply:
Thanks a lot!!